<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
<ErrorHandler class="org.xml.sax.ErrorHandler" ref-id="1978026173" interface="true">
   <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="org/xml/sax/SAXException.xml"/>
   <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="org/xml/sax/SAXParseException.xml"/>
   <methods>
      <error return-type="void" visibility="public" abstract="true">
         <parameters>
            <parameter parameter-type="org.xml.sax.SAXParseException"/>
         </parameters>
         <throws>
            <throwable throws="org.xml.sax.SAXException"/>
         </throws>
      </error>
      <fatalError return-type="void" visibility="public" abstract="true">
         <parameters>
            <parameter parameter-type="org.xml.sax.SAXParseException"/>
         </parameters>
         <throws>
            <throwable throws="org.xml.sax.SAXException"/>
         </throws>
      </fatalError>
      <warning return-type="void" visibility="public" abstract="true">
         <parameters>
            <parameter parameter-type="org.xml.sax.SAXParseException"/>
         </parameters>
         <throws>
            <throwable throws="org.xml.sax.SAXException"/>
         </throws>
      </warning>
   </methods>
</ErrorHandler>